cocos/core/gfx/webgl/webgl-commands.ts

```diff
@@ -826,7 +826,9 @@ export function WebGLCmdFuncCreateTexture (device: WebGLDevice, gpuTexture: IWeb
             errorID(9100, maxSize, device.capabilities.maxTextureSize);
         }
 
-        if (!device.extensions.WEBGL_depth_texture && FormatInfos[gpuTexture.format].hasDepth) {
+        // @ts-expect-error(2341) friend class access
+        const { _textureExclusive: textureExclusive } = device;
```

Comment:
Just add a public getter for the property (since it's child-class-only), don't need to do this here